game-core/src/main/java/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -148,11 +146,11 @@ private void createAndSaveStats(final boolean showPhaseStats) {
     chooser.setFileSelectionMode(JFileChooser.FILES_ONLY);
     final File rootDir = new File(SystemProperties.getUserDir());
     final int currentRound = gameData.getCurrentRound();
-    String defaultFileName =
-        "stats_" + dateTimeFormatter.format(LocalDateTime.now()) + "_" + gameData.getGameName() + "_round_"
-            + currentRound + (showPhaseStats ? "_full" : "_short");
-    defaultFileName = FileNameUtils.removeIllegalCharacters(defaultFileName);
-    defaultFileName = defaultFileName + ".csv";
+    final String defaultFileName = FileNameUtils.removeIllegalCharacters(
+        String.format("stats_%s_%s_round_%s%s",
+            dateTimeFormatter.format(LocalDateTime.now()), gameData.getGameName(),
+            currentRound, showPhaseStats ? "_full" : "_short"))
```

Comment:
Can the underscore in `_full` and `_short` be extracted to the format string?

Reply:
Jup, wanted to do that originally, but forgot it